### ISSUE TITLE
Limit which services are put in template

### ISSUE THREAD

**Issue by emmetog** [shufo/nginx-consul-template#3]
I've been playing around with this container and it looks great! The docs are pretty sparse though.

In particular I would like to know how to limit which services appear in the template. For example I have a consul server on a host which has different containers; a few web containers and a few databases for example. I don't want the mysql services to be put in the template, since nginx won't be routing traffic to the database. I only want certain web containers to be in the template.

Can this be done? If so, could you improve the docs?

**Comment by shufo** [comment]
@emmetog 

Yes, you can exclude a service from template by adding a service name to consul template file like this

`nginx.conf.ctmpl`

``` diff
-{{ range $service := services }} {{ if ne $service.Name "consul" }}
+{{ range $service := services }} {{ if ne $service.Name "consul" }} {{ if ne $service.Name "mysql" }}
```

https://github.com/shufo/nginx-consul-template/blob/master/nginx.conf.ctmpl#L37

I'll add a description for this!
